geotrek/altimetry/tests/test_models.py

```diff
@@ -10,7 +10,7 @@
 
 class AltimetryMixinTest(TestCase):
     def test_get_elevation_chart_none(self):
-        trek = TrekFactory.create(no_path=True, published=True)
+        trek = TrekFactory.create(published=True)
```

Comment:
Si on ne précise pas le paramètre path ça fait quoi ? Ça ne crée aucun path ou bien ça crée un path par défaut ? Dans ce deuxième cas, pour ne pas changer le test, est-ce qu'il ne faut pas préciser le paramètre `path=[]`

Reply:
oui il faut mettre paths=[]

Reply:
paths is None du coup => PathAggregation